### DISCUSSION TITLE
Ingestion from s3/swift?

### DISCUSSION THREAD

**Discussion by sirdatta** [openobserve/openobserve#2832] [Q&A]
Is it possible to ingest year old syslog's from s3 on the fly?

**Comment by prabhatsharma** [comment]
not sure what you mean by on the fly., but OpenObserve cannot read existing syslog files directly. You will need to ingest them in OpenObserve.

**Comment by sirdatta** [comment]
Ingest from S3, may be forwarding it via log forwarder Fluentbit or Vector
